src/ContaoManager/Plugin.php

```diff
@@ -28,9 +29,15 @@ class Plugin implements BundlePluginInterface
      */
     public function getBundles(ParserInterface $parser)
     {
+        $loadAfter = [ContaoCoreBundle::class, 'conditionalformfields', 'leads'];
+
+        if (class_exists(Terminal42LeadsBundle::class)) {
```

Comment:
This isn't actually necessary. Any classes that do not exist will be ignored by the Contao Manager Plugin :)

Reply:
No, these are just strings.

Reply:
I removed the check an tested it, seems to work. Lessen learned :)